crates/sui-protocol-config/src/lib.rs

```diff
@@ -611,7 +611,7 @@ impl ProtocolConfig {
                 max_tx_size: Some(64 * 1024),
                 max_tx_in_batch: Some(10),
                 max_modules_in_publish: Some(128),
-                max_arguments: Some(128),
+                max_arguments: Some(512),
```

Comment:
@oxade @sadhansood @asonnino 
Another one I had to bump for benchmarks. I know we will probably have to bring this back down, but if we could leave it for now so this could get in that would be great :)